README.md

```diff
@@ -40,9 +47,52 @@ Here are some key features of YuniKorn.
 - Customized resource types (like GPU) scheduling support.
 - Rich placement constraints support.
 - Automatically map incoming container requests to queues by policies. 
-- Node partition: partition cluster to sub-clusters with dedicated quota/ACL management. 
+- Node partition: partition cluster to sub-clusters with dedicated quota/ACL management.
+
+### Integration with K8s:
+
+`yunikorn-k8shim` nicely integrated with K8s, supported features like: 
```

Comment:
Not a proper english sentence when I read it.
The k8shim provides the integration for K8s scheduling and `supported features include` is better